commands/host.vegman

```diff
@@ -63,6 +63,71 @@ function cmd_power_status {
   ipmitool power status
 }
 
+# @doc cmd_virtualmedia
+# Access Virtual Media
+function cmd_virtualmedia {
+  subcommand "$@"
+}
+
+# @sudo cmd_virtualmedia_mount admin
+# @doc cmd_virtualmedia_mount
+# Mount a virtual media
+#  --usb - the USB interface type
+#  --usb-ro - the USB interface type (read-only)
+#  --hdd - the HDD interface type
+#  --cdrom - the CDROM interface type (read-only)
```

Comment:
I guess this must go as `the CD/DVD/BD-ROM interface type (read-only), based on file size`. Will that fit?

Reply:
Yes, this more accurately reflects the functionality of the "--cdrom" flag.

Reply:
Fixed.